scripts/installer/install.sh

```diff
@@ -435,6 +441,8 @@ EOF
     ) > "$bisdn_linux_mnt/lib/systemd/network/90-enp.link"
 fi
 
+platform_setup
```

Comment:
`platform_setup` should either come before all the automated/default setup or after, but not in between (both have their merits).

Reply:
Moved call down.